src/shared/components/case-editor/case-edit-page/case-edit-page.html

```diff
@@ -18,7 +18,7 @@ <h1 class="heading-h1 error-summary-heading" id="edit-case-event_error-summary-h
 </div>
 <!-- Event error heading and error message to be displayed if there are specific error details -->
 <div *ngIf="error && error.details" class="error-summary" role="group" aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
-  <h3 class="heading-h3 error-summary-heading" id="edit-case-event_error-summary-heading">
+  <h3 class="heading-h3 error-summary-heading" id="edit-case-event_error-summary-heading-3">
```

Comment:
What purpose does the `-3` serve? Horrible name for an id BTW, but we inherited that.

Reply:
One for @andywilkinshmcts, I think. Not sure why this change was made...